calico/felix/devices.py

```diff
@@ -135,6 +150,16 @@ def configure_interface_ipv6(if_name, proxy_target):
                            "proxy", str(proxy_target), "dev", if_name])
 
 
+def _read_proc_sys(name):
+    with open(name, "rb") as f:
+        return f.read()
```

Comment:
I think it would be nice to do .strip() here, so that callers don't have to.  Are there any downsides to that?